docs/troubleshooting.md

```diff
@@ -28,6 +28,20 @@ In the docker GUI go to the "Preferences" pane, then the "Advanced" tab and move
 
 ![Docker Advanced Settings](./assets/docker-gui-advanced.png)
 
+## Configuration updates when using ElasticSearch on Local Environment
```

Comment:
Because this is troubleshooting try to think about the questions someone might be asking themselves or searching for when writing headings eg. "Services failing to start or randomly stopping"